docs/developer-guide/nodejs-api.md

```diff
@@ -310,6 +310,8 @@ var report = cli.executeOnText("var foo = 'bar';", "foo.js");
 
 The `report` returned from `executeOnText()` is in the same format as from `executeOnFiles()`, but there is only ever one result in `report.results`.
 
+This function will return no errors or warnings if filename provided in the optional 2nd parameter matches a file that is configured to be ignored.  Pass `true` as the optional 3rd parameter to have a warning generated when this happens.
```

Comment:
@robertlevy Here is wording for how I interpreted the cause-and-effect relationship, but not sure if it is correct:

If a filename in the optional second parameter matches a file that is configured to be ignored, then this function returns no errors or warnings. To return a warning instead, call the method with `true` as the optional third parameter.

If the preceding is unusable, here are incremental suggestions about wording in the original:
* if the filename in the optional second parameter
* Call the method with `true` as the optional third parameter